recipes/sass_setup.rb

```diff
@@ -0,0 +1,9 @@
+remove_file 'app/assets/stylesheets/application.css'
+create_file 'app/assets/stylesheets/application.scss'
```

Comment:
Do we want to keep any defaults in this file copied over from the one we're deleting?